restaurant/searchitems.js

```diff
@@ -0,0 +1,32 @@
+function searchMenu() {
+  var menu = JSON.parse(localStorage.getItem("menu"));
+  var searchValue = document.getElementById("search-menu").value.toLowerCase();
+  var menu_container = document.getElementById("menu-container");
+
+  menu_container.innerHTML = "";
+  console.log(menu_container);
+
+  for (let i = 0; i < menu.length; i++) {
+    var itemName = menu[i].Name.toLowerCase();
```

Comment:
don't use var , make use of let and const
Reference : https://www.freecodecamp.org/news/differences-between-var-let-const-javascript/#:~:text=If%20you're%20just%20starting,shouldn't%20use%20var%20anymore.